src/ansys/geometry/core/_grpc/_services/base/repair_tools.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (C) 2023 - 2025 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Module containing the repair tools service implementation (abstraction layer).
+
+This module defines an abstract base class for a gRPC-based repair tools service.
+The class provides a set of abstract methods for identifying and repairing various
+geometry issues, such as split edges, extra edges, duplicate faces, and more.
+"""
+
+from abc import ABC, abstractmethod
+
+import grpc
+
+
+class GRPCRepairToolsService(ABC):
+    """Abstract base class for gRPC-based repair tools service.
+
+    Defines abstract methods for identifying and repairing geometry issues in a model.
+    """
+
+    def __init__(self, channel: grpc.Channel):
+        """Initialize the gRPC repair tools service.
+
+        Parameters
+        ----------
+            channel (grpc.Channel): The gRPC channel used to communicate with the service.
+        """
+        pass  # pragma: no cover
+
+    @abstractmethod
+    def find_split_edges(self, **kwargs):
```

Comment:
You are missing the return typehints throughout the module - please adapt
```suggestion
    @abstractmethod
    def find_split_edges(self, **kwargs) -> dict:
```